image/cli-base/install/install-rclone.sh

```diff
@@ -11,3 +11,7 @@ cp ./rclone-*-linux-amd64/rclone /usr/local/bin/
 rclone version
 
 rm -rf rclone-*
+
+#s390x not supported
+#[root@aca72e69ab39 rclone-v1.65.2-linux-mipsle]# rclone version
+ #bash: /usr/local/bin/rclone: cannot execute binary file: Exec format error
```

Comment:
How will the backup/restore function work?

Reply:
Initially we are going to state limited support for running on s390x ... the README in this repo needs to be updated with the details of what restrictions are in place in the s390x image ... e.g. we're expecting that neither aws or rclone will be included on the s390x base image; and when we build the s390x cli image we will update it's README to document the features of the cli that are not supported on s390x.